exchange/exchange.go

```diff
@@ -469,7 +479,8 @@ func (e *exchange) buildBidResponse(ctx context.Context, liveAdapters []openrtb_
 	bidResponse.SeatBid = seatBids
 
 	if bidResponseExt == nil {
-		bidResponseExt = e.makeExtBidResponse(adapterBids, adapterExtra, bidRequest, resolvedRequest, errList)
+		debugInfo := ctx.Value(DebugContextKey("debugInfo"))
+		bidResponseExt = e.makeExtBidResponse(adapterBids, adapterExtra, bidRequest, resolvedRequest, debugInfo != nil, errList)
```

Comment:
This is kinda what I was saying above, passing in `debugInfo != nil` looks kinda odd here IMO and also makes it a little difficult to read the code.

Reply:
I don't think we need the overhead of setting a `false` value into the `Context`. If we simply consider `debugInfo` to be false when not set, we save time and space.
In regards to readability, do you think it improves it if we do:
```
debugInfo := ctx.Value(DebugContextKey("debugInfo")) != nil
bidResponseExt = e.makeExtBidResponse(adapterBids, adapterExtra, bidRequest, resolvedRequest, debugInfo, errList)
```